frontend/src/components/EmployeeDetails.js

```diff
@@ -0,0 +1,14 @@
+const EmployeeDetails = ({employee}) => {
+    
+    return (
+        <div className="employee-details">
+            <strong><p>{employee.imie} {  } {employee.nazwisko}</p></strong>
+            <p>Data dołączenia do rejestru pracowników: {employee.createdAt}</p>
```

Comment:
"Data dołączenia do rejestru pracowników" brzmi dziwnie, jakby to pracownik podejmował jakieś działanie "dołączenia" - tego typu sformuowanie bardziej pasuje do zdania "Data dołączenia Polski do NATO" przykładowo. Szczerze zmieniłbym to na "Data dodania do rejestru", po prostu.